hooks/playbooks/ironic_enroll_nodes.yml

```diff
@@ -7,6 +7,7 @@
     _namespace: openstack
     _ironic_node_name_prefix: ironic-
     _ironic_network_interface: flat
+    _ironic_network_physical_network: ironic
```

Comment:
you don't really need to set a variable for that, since it can't be overridden - you can just pass the value as a string in the dict bellow. Or am I missing some other occurrences?

Reply:
When defining the job it should be possible to override - 

For example:
```
  - name: 15 Run nova-manage cell_v2 discover_hosts
    type: playbook
    source: nova_manage_discover_hosts.yml
    extra_vars:
      _cell_conductors: nova-cell1-conductor-0
``

Reply:
hmm ok. right, that's a playbook level var, didn't realize.